velox/functions/lib/aggregates/DecimalAggregate.h

```diff
@@ -338,4 +338,37 @@ class DecimalAggregate : public exec::Aggregate {
   DecodedVector decodedPartial_;
 };
 
+/* This method is used to correct the sum result calculated using
+ * addWithOverflow. In the addWithOverflow method, when summing two int128_t
```

Comment:
Should we move this function next to addWithOverflow?

Use /// for comments on public API.

Reply:
This function was initially placed after `addWithOverflow`, but since it is only relevant to aggregation, it was moved here. However, I believe it would be more appropriate to place it in `DecimalUtil` since it serves as a supplementary method for `addWithOverflow`.

Reply:
I agree. Let's move it next to addWithOverflow. Let's also make sure tests for both functions are in the same file.

Reply:
Let's move it back to `DecimalUtil`. cc @rui-mo